markdown/ISD-NATIONAL-STATS-REPORT.Rmd

```diff
@@ -59,7 +59,16 @@ trend_data <- read_csv(here("data",
                             "output",
                             paste0(pub_date(end_date = end_date,
                                             pub = "current"),
-                                   "_trends-data-level2.csv"))) %>%
+                                   "_trends-data-level2.csv")),
+                            col_types = cols(
+                         quarter = col_double(),
+                         quarter_short = col_character(),
+                         quarter_full = col_character()
+                       )) %>%
+  filter(time_period == "Quarter") %>% 
+  select(hb, location, location_name, agg_label, quarter, quarter_short,
```

Comment:
If you are deleting a variable here you can do that by putting a minus in front of the name. Might be a bit neater than typing them all out!

Reply:
Thanks David, agree this would be neater but I'm altering the order of variables as well as removing unnecessary ones so will have to leave it as is 😞